examples/wallet_smart_contract/src/main.sw

```diff
@@ -1,19 +1,22 @@
-// NOTE: Storage is a work in progress (see
-// https://github.com/FuelLabs/sway/pull/646), but once it is implemented,
-// declaring storage should look like this.
-
 contract;
 
-use std::*;
-use std::address::Address;
-use std::assert::assert;
+use std::{
+    address::Address, 
+    assert::assert, 
+    chain::auth::{AuthError, Sender, msg_sender}, 
+    constants::NATIVE_ASSET_ID, 
+    context::{call_frames::msg_asset_id, msg_amount}, 
+    contract_id::ContractId, 
+    panic::panic, 
+    result::*, 
+    token::transfer_to_output
```

Comment:
Does the formatter remove the trailing `,` here? If so, file an issue to make it add/keep the trailing comma.

Reply:
Actually, the trailing `,` is not currently allowed by the parser. I'll file an issue.